pyorerun/biorbd_components/mesh.py

```diff
@@ -50,17 +51,19 @@ def _set_rerun_mesh3d(self):
             )
         else:
             self.__rerun_mesh = rr.Mesh3D(
-                vertex_positions=self.__mesh.vertices,
+                vertex_positions=transformed_trimesh.vertices,
```

Comment:
You missed the self.__mesh in the first if case.

Reply:
Please, let me know if what I did is ok